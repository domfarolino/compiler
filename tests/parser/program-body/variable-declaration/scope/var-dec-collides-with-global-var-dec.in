program dominic is
  global integer dominic;

  procedure function()
    integer dominic;
  begin
    dominic := 1000;
  end procedure;

begin
end program.
