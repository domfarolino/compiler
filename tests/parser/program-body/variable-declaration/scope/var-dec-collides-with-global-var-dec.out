[1;31mLine 5: Symbol 'dominic' already exists, cannot redeclare
Line 5: Error parsing program body
[0m